funsor/gaussian.py

```diff
@@ -36,26 +36,22 @@ def _trace_mm(x, y):
     """
     assert x.dim() >= 2
     assert y.dim() >= 2
-    return (x * y).sum([-1, -2])
+    return (x * y).sum((-1, -2))
 
 
 def cholesky(u):
     """
     Like :func:`torch.cholesky` but uses sqrt for scalar matrices.
     Works around https://github.com/pytorch/pytorch/issues/24403 often.
     """
-    if u.size(-1) == 1:
-        return u.sqrt()
-    return u.cholesky()
+    return ops.cholesky(u)
```

Comment:
You can now remove `cholesky` and `cholesky_inverse` from `gaussian.py`

Reply:
I'll remove those in #296 PR. For now, it is required for internal math of gaussian and some other files too.